Shubham_Sharma/JWT Assignment/UserAuthorization/UserAuthorization/Controllers/AdminController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace UserAuthorization.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AdminController : ControllerBase
+    {
+        DbOperations.IDbOperations _idbo;
+        public AdminController(DbOperations.IDbOperations idbo)
+        {
+            _idbo = idbo;
+        }
+
+        [Route("getAllUsers")]
+        [HttpPost]
+        public IActionResult getAll([FromBody] CustomModels.CustomEmployee customEmployee)
```

Comment:
Route should not contain http verbs in them . This is not restful practice

Reply:
Fixed the issue here......